src/na/na_ofi.c

```diff
@@ -3995,7 +3995,7 @@ na_ofi_get_uri(const struct na_ofi_fabric *na_ofi_fabric,
 
     /* Remove unnecessary "://" prefix from string if present */
     if (strstr(fi_addr_str, "://")) {
-        strtok_r(fi_addr_str, ":", &fi_addr_str_ptr);
+        fi_addr_str_ptr = strstr(fi_addr_str, ":") + 1;
         rc = strncmp(fi_addr_str_ptr, "//", 2);
```

Comment:
that's a good point and we could simplify the code even more actually since we don't need a second strstr() and the following strncmp, could you please do that instead ?

Reply:
Sure, I simplified the code further.

Reply:
Thanks!